go-controller/pkg/node/node_ip_handler.go

```diff
@@ -0,0 +1,118 @@
+package node
```

Comment:
I can't seem to remember the correct incantation for this, but doesn't this mean that this will run on windows nodes too?

Reply:
yeah I think so, but we don't support running on windows. I think we gave up on that a while ago. It shouldn't be hard to fix this though. I'll take a quick look on the next patch set.

Reply:
should be very easy. keep the struct here, move your implementation to `node_ip_handler_linux.go` and add an empty one for windows.

Reply:
or you can use build tags, both do the same
```
// +build linux
```

xref https://dave.cheney.net/2013/10/12/how-to-use-conditional-compilation-with-the-go-build-tool